public/app/features/alerting/unified/components/receivers/ReceiversTable.tsx

```diff
@@ -0,0 +1,83 @@
+import { useStyles } from '@grafana/ui';
+import { receiverTypeNames } from 'app/plugins/datasource/alertmanager/consts';
+import { AlertManagerCortexConfig } from 'app/plugins/datasource/alertmanager/types';
+import { capitalize } from 'lodash';
+import React, { FC, useMemo } from 'react';
+import { useUnifiedAlertingSelector } from '../../hooks/useUnifiedAlertingSelector';
+import { getAlertTableStyles } from '../../styles/table';
+import { ActionButton } from '../rules/ActionButton';
+import { ActionIcon } from '../rules/ActionIcon';
+import { ReceiversSection } from './ReceiversSection';
+
+interface Props {
+  config: AlertManagerCortexConfig;
+}
+
+export const ReceiversTable: FC<Props> = ({ config }) => {
+  const tableStyles = useStyles(getAlertTableStyles);
+
+  const grafanaReceiverTypes = useUnifiedAlertingSelector((state) => state.grafanaReceiverTypes);
+
+  const receivers = useMemo(
+    () =>
+      config.alertmanager_config.receivers?.map((receiver) => ({
```

Comment:
what is this config object that is using snake case?

Reply:
cortex alertmanager config: https://cortexmetrics.io/docs/api/#alertmanager-configs

Reply:
Ok, why is it on the global instanse config object ? Is this something you configure per Grafana server in the ini file?

Reply:
Ah maybe it's not the global instance config (reviewing from mobile )

Reply:
Yup, this is not related to the grafana instance config. This is prometheus alert manager config that is managed via API